testfiles/RadLoTempHydrHeatCoolMultiZone.idf

```diff
@@ -0,0 +1,2195 @@
+! RadLoTempHydrHeatCoolMultiZone.idf
```

Comment:
Here's what is in the error file for this:
```
   ** Warning ** SetUpZoneSizingArrays: Requested Sizing for Zone="EAST ZONE", Zone is not found in the Controlled Zones List
   ** Warning ** SetUpZoneSizingArrays: Requested Sizing for Zone="EAST ZONE", Zone has no thermostat (ref: ZoneControl:Thermostat, et al)
   ** Warning ** SetUpZoneSizingArrays: Requested Sizing for Zone="NORTH ZONE", Zone is not found in the Controlled Zones List
   ** Warning ** SetUpZoneSizingArrays: Requested Sizing for Zone="NORTH ZONE", Zone has no thermostat (ref: ZoneControl:Thermostat, et al)
   ** Warning ** Not all surfaces in same zone for ZoneHVAC:LowTemperatureRadiant:SurfaceGroup = SURFACESINTHREEZONES
   ** Warning ** Not all surfaces in same zone for ZoneHVAC:LowTemperatureRadiant:SurfaceGroup = SURFACESINTHREEZONES
   ** Warning ** A surface referenced in a Low Temperature Radiant System is not in same zone as the radiant system itself
   **   ~~~   ** Surface = ZN002:FLR001
   **   ~~~   ** Surface in Zone=EAST ZONE Radiant System in Zone = WEST ZONE
   **   ~~~   ** Occurs in Low Temperature Radiant System = WEST ZONE RADIANT FLOOR
   **   ~~~   ** If this is intentionally a radiant system with surfaces in more than one thermal zone,
   **   ~~~   ** then ignore this warning message.  Otherwise, check the surfaces in this radiant system.
   ** Warning ** A surface referenced in a Low Temperature Radiant System is not in same zone as the radiant system itself
   **   ~~~   ** Surface = ZN003:FLR001
   **   ~~~   ** Surface in Zone=NORTH ZONE Radiant System in Zone = WEST ZONE
   **   ~~~   ** Occurs in Low Temperature Radiant System = WEST ZONE RADIANT FLOOR
   **   ~~~   ** If this is intentionally a radiant system with surfaces in more than one thermal zone,
   **   ~~~   ** then ignore this warning message.  Otherwise, check the surfaces in this radiant system.
```
1. The Sizing:Zone objects should be deleted for East Zone and North Zone.
2. That's a lot of words for a warning that might be ignorable. 
2a. This one should only fire once per surfacegroup: `** Warning ** Not all surfaces in same zone for ZoneHVAC:LowTemperatureRadiant:SurfaceGroup = SURFACESINTHREEZONES`
2b. Since this is a potentially OK condition, how about wrapping `if (DisplayExtraWarnings)` around the surface-by-surface warnings for not in the same zone.
2c. These lines could move to the surface group warning:
```
   **   ~~~   ** If this is intentionally a radiant system with surfaces in more than one thermal zone,
   **   ~~~   ** then ignore this warning message.  Otherwise, check the surfaces in this radiant system. Use Output:Diagnostics, DisplayExtraWarnings for more details.
```

Reply:
@mjwitte Looks great, Mike!  Thanks for making that change, for your review, and for getting this merged in!! 😀👍